README.md

```diff
@@ -102,9 +106,23 @@ These are the supported vector properties:
 
 #### points
 
-The `points` setting defines a list of points to be drawn on the
-board. It is optional. Each point should be defined as an object with
-`name` and `coords` properties.
+The `points` setting defines a list of points to be either just drawn
+on the board for reference, or to be placed by the student.  Points can
+have the following properties:
+
+* `name` (required): The unique name of the point, used in checks and
+  internally to identify the point.
+* `coords` (required): An array of length 2 with the (initial)
+  coordinates of the point.
+* `fixed`: Whether the point should be drawn in a fixed location or
+  placed by the student.  Defaults to `true`.
```

Comment:
@smarnach ... and reading this I have no idea what `true` stands for. :) Well, actually, since I'm familiar with how this thing works internally I do, but someone else will probably be confused by this.